README.md

```diff
@@ -28,10 +28,10 @@ python -m venv venv
 source venv/bin/activate
 ```
 
-2. Install cruft
+2. Install necessary files
 
 ```sh
-python -m pip install cruft
+python -m pip install cruft packaging ruff black
```

Comment:
Why not just install -r requirements.txt?

Reply:
If you pip install -r requirements then you have to pull down this repo. I wanted to make it plain so that someone (without cloning this repo) could have all the things needed and build from the examples, considering that cruft is going to make them pull from the github repo and not a local version.

Reply:
Ahhh right, didnt think about that. LGTM